lib/services/session.go

```diff
@@ -113,33 +117,33 @@ func MarshalWebSession(ws types.WebSession, opts ...MarshalOption) ([]byte, erro
 		}
 		return utils.FastMarshal(webSession)
 	default:
-		return nil, trace.BadParameter("unrecognized web session version %T", ws)
+		return nil, trace.BadParameter("unrecognized web session version %T", s)
 	}
 }
 
 // MarshalWebToken serializes the web token as JSON-encoded payload
-func MarshalWebToken(token types.WebToken, opts ...MarshalOption) ([]byte, error) {
+func MarshalWebToken(t types.WebToken, opts ...MarshalOption) ([]byte, error) {
 	cfg, err := CollectOptions(opts)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+
 	version := cfg.GetVersion()
```

Comment:
@a-palchikov Is there a reason you chose to use the `Version` marshal option, rather than using `token.GetVersion` or switching on the token type?

Reply:
This is a bug on my side - feel free to modify to use the version from the metadata block.